turbo.json

```diff
@@ -20,6 +20,9 @@
     },
     "lint": {
       "dependsOn": ["format", "^build"]
+    },
+    "attw": {
+      "dependsOn": ["^build"]
```

Comment:
This is incorrect, see https://github.com/connectrpc/connect-query-es/pull/184#discussion_r1341713169

Reply:
Yep, fixed.